template/README.md.jinja

```diff
@@ -21,6 +21,16 @@ If you use {{ PackageName }}.jl in your work, please cite using the reference gi
 
 If you want to make contributions of any kind, please first that a look into our [contributing guide directly on GitHub](docs/src/90-contributing.md) or the [contributing page on the website](https://{{ PackageOwner }}.github.io/{{ PackageName }}.jl/dev/contributing/).
 
+### Contributors
```

Comment:
```suggestion
{% if AddAllcontributors %}
### Contributors

<!-- markdown-link-check-disable -->
```
You need the conditional, but I also added this extra directive to avoid link issues.